Add "removedb" command to Geth

tgerring opened the pull request:
Removes the state and blockchain databases

Closes #1010

obscuren commented on cmd/geth/main.go:
```diff
@@ -543,6 +548,16 @@ func exportchain(ctx *cli.Context) {
 	return
 }
 
+func removeDb(ctx *cli.Context) {
```
Please add a `are you sure [y/N]` thingy

tgerring commented:
Example interaction with confirmation:

```
$ geth removedb
Remove local databases? [y/N] 
Operation aborted
$ geth removedb
Remove local databases? [y/N] N
Operation aborted
$ geth removedb
Remove local databases? [y/N] Y
Removing chain and state databases...
Removed in 104.922µs
$ geth removedb
Remove local databases? [y/N] y
Removing chain and state databases...
Removed in 55.145µs
```
